pkg/authenticator/version.go

```diff
@@ -6,11 +6,11 @@ import "fmt"
 // of the authn-k8s-client
 var Version = "0.19.1"
 
-// Tag field denotes the specific build type for the client. It may
+// VersionSuffix field denotes the specific build type for the client. It may
 // be replaced by compile-time variables if needed to provide the git
 // commit information in the final binary
-var Tag = "dev"
+var VersionSuffix = "Dev"
 
 // FullVersionName is the user-visible aggregation of version and tag
 // of this codebase
-var FullVersionName = fmt.Sprintf("%s-%s", Version, Tag)
+var FullVersionName = fmt.Sprintf("%s%s", Version, VersionSuffix)
```

Comment:
i would call it `TagSuffix` and add a comment saying that in fixed versions we don't want the tag to be present.
Also, it should be `var TagSuffix = "-Dev"`.

Reply:
Changed accordingly